.gitignore

```diff
@@ -1,54 +1,35 @@
-# --------------------
-# OSX Files
-# --------------------
+### SublimeText ###
+*.sublime-workspace
+
+### OSX ###
 .DS_Store
 .AppleDouble
 .LSOverride
 Icon
+
+# Thumbnails
 ._*
+
+# Files that might appear on external disk
 .Spotlight-V100
 .Trashes
 
-# --------------------
```

Comment:
I wanted it to match our component boilerplate .gitignore for consistency, but do you think I should instead add these lines to our boilerplate?